codeforamerica/mdpilot
# Implements Need and Program Pages

Implements Need and Program Pages:

To test:
1. Select A Need (if you select either of the top two options, the next window will show both options with the need you selected prechecked).
2. Go Back - Select all needs (except not sure) and confirm that in the next page, 4 options appear that are all checked.
3. Go back. Select "Not Sure". Notice it act's as a "None of the Above". Go to the next page and note that all options are listed but none are checked.

Play around with moving forward to the signpost, and then back: Does the behavior make sense to you?

Once you feel good about the need, check that the pdf form is properly filled out:
If you select SNAP, the snap option on the first page will be checked.
If you select something aside from SNAP, Cash Assistance will be checked.
